.github/workflows/frontend.yml

```diff
@@ -124,10 +124,19 @@ jobs:
           github-token: ${{ steps.token.outputs.token }}
 
       - name: tsc
+        id: tsc
         if: steps.dependencies.outcome == 'success' && needs.files-changed.outputs.frontend == 'true'
         run: |
           set -o pipefail
-          yarn tsc -p config/tsconfig.build.json --diagnostics | tee /tmp/typescript-monitor.log && yarn run ts-node .github/workflows/scripts/monitor-typescript.ts
+          yarn tsc -p config/tsconfig.build.json --diagnostics --generateTrace /tmp/trace | tee /tmp/typescript-monitor.log
+
+      - name: monitor-tsc
+        continue-on-error: true
```

Comment:
Under what conditions does `yarn run ts-node` fail to execute?

Reply:
I'm not too certain but this was done as a precaution to prevent the workflow from failing as this is purely for analytics.